### PR TITLE
Feature/ant drawer/nikid

### PR DESCRIPTION
**Pull request by ndossett**

### CODE REVIEW

**Review by c-melchor — COMMENTED** [review]
Looks good, just need to get rid of commented out code in oktaConfig.js file.

**Comment by ndossett** [comment]
> Looks good, just need to get rid of commented out code in oktaConfig.js file.

I didn't see commented out code in oktaConfig, however I removed the console.log

**Review by c-melchor — APPROVED** [review]
Looks ready to go with console log removed.

File: src/components/pages/CaseTable/CaseTable.js [hunk 1]
```diff
@@ -604,17 +607,19 @@ export default function CaseTable(props) {
...
+        <DataGrid
+          rows={searching ? filter(caseData) : caseData}
+          columns={columns}
+          className={classes.grid}
+          loading={caseData ? false : true}
+          checkboxSelection={true}
+          onSelectionModelChange={onCheckboxSelect}
+          showCellRightBorder={true}
+          disableColumnMenu={true}
+          components={{ Toolbar: Toolbar }}
```
Comment (crystal-csete) [inline src/components/pages/CaseTable/CaseTable.js:656]: Nice color matching.

File: src/components/pages/SideDrawer/SideDrawerData.js [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
   {
     title: 'Saved Cases',
     path: '/saved-cases',
-    icon: <BookmarksIcon />,
+    icon: <BookmarksOutlinedIcon />,
   },
   {
     title: 'Saved Judges',
     path: '/saved-judges',
-    icon: <BookmarkIcon />,
```
Comment (crystal-csete) [inline src/components/pages/SideDrawer/SideDrawerData.js:32]: These changes look good to me! Clean code!

File: src/components/pages/SideDrawer/SideDrawerAnt.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        placement="left"
+        closable={false}
+        onClose={onClose}
+        visible={visible}
+      >
+        <p>Cases</p>
+        <p>Judges</p>
+        <p>Upload Case</p>
+        <p>Saved Cases</p>
+        <p>Saved Judges</p>
```
Comment (crystal-csete) [inline src/components/pages/SideDrawer/SideDrawerAnt.js:31]: Clean code, looks great!

File: src/components/pages/SideDrawer/SideDrawer.js [hunk 4]
```diff
@@ -211,5 +211,5 @@ export default function SideDrawer(props) {
 }
```
Comment (crystal-csete) [inline src/components/pages/SideDrawer/SideDrawer.js:212]: Nice color matching.

**Review by crystal-csete — COMMENTED** [review]
These changes look good to me!

**Review by crystal-csete — APPROVED** [review]

**Pull request merged**